### PR TITLE
Add mini side loading states

### PR DESCRIPTION
**Pull request by chinins**
## Description

This PR adds `MiniSpinnerLoader` core component to replace repetitive code in places where small spinners are used.

**New stuff** ✨

- add `MiniSpinnerLoader`

**Changes** 🏗

- replace `SpinnerLoader` with `MiniSpinnerLoader`
- replace repetitive css with standard css in `MiniSpinnerLoader`

**Deletions** ⚰️

none

## TODO

none

Resolves DEV-227

### CODE REVIEW

File: src/modules/core/components/Preloaders/MiniSpinnerLoader.tsx [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+      {title && (
+        <Heading
+          text={title}
+          textValues={titleTextValues}
+          appearance={titleAppearance}
+        />
+      )}
+      <SpinnerLoader appearance={appearance} />
+      {loadingText && (
+        <span className={loadingTextClassName || styles.loadingText}>
```
Comment (rdig) [inline src/modules/core/components/Preloaders/MiniSpinnerLoader.tsx:41]: I don't think we should have this customizable, but it your choice in the end, so I'm fine if you decide to leave it in.

**Review by rdig — APPROVED** [review]
Very nice work on cleaning this up! :1st_place_medal:

**Review by ArmandoGraterol — APPROVED** [review]
Looks good to me :ok_hand:

File: src/modules/core/components/Preloaders/MiniSpinnerLoader.tsx (hunk 1 above)
Comment (chinins) [reply src/modules/core/components/Preloaders/MiniSpinnerLoader.tsx:41]: In some components the styles are a bit different - like when we have it in a dialog and a couple of other instances. So I prefer to leave it like that.

**Pull request merged**